app/Commands/Stop.php

```diff
@@ -3,25 +3,22 @@
 namespace App\Commands;
 
 use App\Builder\DockerCompose;
-use App\Commands\Traits\HasDynamicArgs;
 
 class Stop extends Command
 {
-    use HasDynamicArgs;
-
     /**
-     * The name of the command.
+     * The signature of the command.
      *
      * @var string
      */
-    protected $name = 'stop';
+    protected $signature = 'stop {--purge}';
```

Comment:
add more description about purge, maybe also explaining in docker language, `down --volumes`

Reply:
also add `--remove-orphans`